js/mojs-player.babel.js

```diff
@@ -400,6 +400,12 @@ class MojsPlayer extends Module {
   */
   _onHideStateChange ( isHidden ) {
     this._props.isHidden = isHidden;
+    const { onToggleHide } = this._o;
+    // skip running callback on init rendering
+    if ( this._hideCount && onToggleHide) {
```

Comment:
Oh I see, this might be confusing sorry. We need it fire on the initial render - we can capture initial state too this way.